assets/models/system/Mixtral-8x7B-v0-1/description.md

```diff
@@ -0,0 +1,49 @@
+# **Model Details**
+
+The Mixtral-8x7B-v0.1 Large Language Model (LLM) is a pretrained generative text model with 7 billion parameters. 
+Mixtral-8x7B-v0.1 outperforms Llama 2 70B on most benchmarks with 6x faster inference.
+
+For full details of this model please read [release blog post](https://mistral.ai/news/mixtral-of-experts/).
+
+## Model Architecture
+
+Mixtral-8x7B-v0.1 is a decoder-only model with 8 distinct groups or the "experts". At every layer, for every token, a router network chooses two of these experts to process the token and combine their output additively. Mixtral has 46.7B total parameters but only uses 12.9B parameters per token using this technique. This enables the model to perform with same speed and cost as 12.9B model.
+
+# **Inference samples**
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation-dolly" target="_blank">text-generation-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation-dolly" target="_blank">text-generation-online-endpoint.sh</a>
+Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-generation" target="_blank">text-generation-batch-endpoint.ipynb</a>| coming soon
+
+
+## **Sample inputs and outputs (for real-time inference)**
```

Comment:
```suggestion
# **Sample inputs and outputs**
```

Reply:
based on recent discussion with Tina. We are going to update few things on the model card. Its documented here as well: [doc](https://microsoftapc-my.sharepoint.com/:w:/g/personal/ayushmishra_microsoft_com/ETjbEy6poUhBsdysFKduH3IBmrdEe2gP2Yzza3p6COAAhA?e=SdAwKu)